src/nanodbc.cpp

```diff
@@ -1823,6 +1835,40 @@ void statement::statement_impl::bind(
 
 void statement::statement_impl::bind_strings(
     short param,
+    const std::vector<string_type>& values,
+    std::size_t elements,
+    const bool* nulls,
+    const string_type::value_type* null_sentry,
+    param_direction direction)
+{
+
+    SQLSMALLINT data_type;
+    SQLSMALLINT param_type;
+    SQLULEN parameter_size;
+    SQLSMALLINT scale;
+    prepare_bind(param, elements, direction, data_type, param_type, parameter_size, scale);
+
+    size_t max_len = 0;
+    for (std::size_t i = 0; i < elements; ++i)
+    {
+        if (values[i].length() > max_len)
+        {
+            max_len = values[i].length();
+        }
+    }
+    // add space for null terminator
+    ++max_len;
+
+    data_[param] = std::vector<string_type::value_type>(max_len * elements, 0);
+    for (std::size_t i = 0; i < elements; ++i)
+    {
+        std::copy(values[i].begin(), values[i].end(), data_[param].data() + (i * max_len));
```

Comment:
I committed a tweak (c1c0d8a7f1d9c6ffe30e1889fe1b2e5d92510266) that will help to avoid Visual C++ C4996 warning here.